### PR TITLE
[server] Remove admin OTS create/use flow

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

No longer needed, as it's super-seeded by the admin-credentials file flow

## Related Issue(s)

* Depends on https://github.com/gitpod-io/gitpod/pull/16760/files
* Part of https://github.com/gitpod-io/gitpod/issues/16223

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/server/src/installation-admin/installation-admin-controller.ts [hunk 1]
```diff
@@ -43,46 +28,6 @@ export class InstallationAdminController {
...
-                if (!this.config.admin.loginKey) {
-                    throw new Error("Cannot handle request");
-                }
-
-                // Unblock the admin-user: it's blocked initially!
-                const user = await this.userDb.findUserById(BUILTIN_INSTLLATION_ADMIN_USER_ID);
-                if (!user) {
-                    throw new Error("Cannot find builtin admin-user");
-                }
-                user.blocked = false;
```
Comment (geropl) [inline components/server/src/installation-admin/installation-admin-controller.ts:63]: @easyCZ How is the admin-user unblocked in the new flow?

Just want to make sure this is not overlooked.

File: components/server/src/user/user-controller.ts [hunk 2]
```diff
@@ -190,38 +190,6 @@ export class UserController {
             }
         });
 
-        router.get(
```
Comment (geropl) [inline components/server/src/user/user-controller.ts:193]: How do we login to a websession with the new credentials? Not set up, yet?

**Review by geropl — APPROVED** [review]
LGTM, comments just for awareness/curiosity

File: components/server/src/user/user-controller.ts (hunk 2 above)
Comment (easyCZ) [reply components/server/src/user/user-controller.ts:193]: It's implemented above this diff - https://github.com/gitpod-io/gitpod/pull/16761/files/0ce214c49a96e782b23f6079c873f01218700d75#diff-0fda9115576b7e1168858f5d4f8869e3120c3a8364cc8f43335037ebb78f6734L145

**Pull request merged**